### DISCUSSION TITLE
Stress Implementation does not seem to be working

### DISCUSSION THREAD

**Discussion by tgmaxson** [mir-group/pair_nequip_allegro#16] [General]
When I attempt to use the stress implementation I get this error.

```
Traceback of TorchScript, original code (most recent call last):
  File "/home/tgmaxson/nequip/nequip/nn/_rescale.py", line 162, in forward
            if self.has_scale:
                for field in self.scale_keys:
                    v = data[field]
                        ~~~~~~~~~~~ <--- HERE
                    data[field] = v * self.scale_by.expand(v.shape)
            if self.has_shift:
RuntimeError: KeyError: stress
```

I am using the stress branch of pair_allegro so I am unsure what would cause this.  I am also using the development branch of nequip (also tried the main branch) and the main branch of allegro.  The model builders are set as following.

```
model_builders:
- allegro.model.Allegro
- PerSpeciesRescale
- StressForceOutput
- RescaleEnergyEtc
```

**Comment by Linux-cpp-lisp** [comment]
Hi @tgmaxson ,

Can you please pull the `develop` branch and rebuild your model? I _think_ I fixed this there.

(You can use `load_model_state` in a config to `nequip-deploy` to avoid retraining.)

**Comment by tgmaxson** [comment]
Can you clarify which combination of versions should work?

**Reply by Linux-cpp-lisp** [reply]
Good point :laughing: 

`pair_allegro` on `stress`
`nequip` on `develop`
`allegro` on `main` or `develop`
